### DISCUSSION TITLE
follow point shows before hitcircle

### DISCUSSION THREAD

**Discussion by Hendshues2** [ppy/osu#20187] [Q&A]
on the right side (i marked on the video) a follow point comes in and than another one, this has never happend to me in stable and i think it's a bug cus it has happend multiple times in lazer and i keep missreading because of it

skin: https://www.mediafire.com/file/1ufifsmo99q3y6u/-_Seoul_v10_%2528spectator_cursor%2529_edited_skin_ini_folder.osk/file
the map: https://osu.ppy.sh/beatmapsets/773014#osu/2151271
(i also realised this happens with any skin that has followpoints, not specifically mine)

https://user-images.githubusercontent.com/47485870/189063235-a3b4856a-f416-4f91-8b91-701c1820d96f.mp4

**Comment by peppy** [comment]
Can you confirm this happens using the default skin (classic or otherwise)? This looks similar to https://github.com/ppy/osu/discussions/19807, and I believe it is likely related to the animation in your custom skin playing at an offset, but have not been able to confirm yet.

**Reply by Hendshues2** [reply]
it's hard to test because it doesn't always happen, it's random for me
